test/views/coronavirus_local_restrictions_test.rb

```diff
@@ -0,0 +1,90 @@
+require "test_helper"
+require "gds_api/test_helpers/mapit"
+
+class CoronavirusLocalRestrictionTest < ActionView::TestCase
+  include GdsApi::TestHelpers::Mapit
+
+  test "renders postcode match tier 4" do
+    render_tier_results(4)
+
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_pretext")
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_tier_label")
+    assert_includes rendered, area
+  end
+
+  test "renders postcode match tier 3" do
+    render_tier_results(3)
+
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_pretext")
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_three.heading_tier_label")
+    assert_includes rendered, area
+  end
+
+  test "renders postcode match tier 2" do
+    render_tier_results(2)
+
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_pretext")
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_tier_label")
+    assert_includes rendered, area
+  end
+
+  test "renders postcode match tier 1" do
+    render_tier_results(1)
+
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_pretext")
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_one.heading_tier_label")
+    assert_includes rendered, area
+  end
+
+  def area
+    "Tattooine"
+  end
+
+  def postcode
+    "E1 8QS"
+  end
```

Comment:
This method doesn't seem to be used outside of `render_tier_results` - I wonder if it can either be just hardcoded in there until something else is needed or be set as an optional argument to the method, for example `render_tier_result(postcode: "E1 8QS")`

Reply:
Thanks, I refactored this.